src/main/java/proto/objects.proto

```diff
@@ -0,0 +1,30 @@
+/*
+Instead of encoding all data in one proto, we will instead preface the proto
+with a header field containing the number of total bytes in the proto. This will
+be a configurable value that will be computed where the biggest possible Data
+division below the maximum threshold is used. This will allow for easy chunking
+when reading in messages etc.
+*/
+syntax = "proto3";
+
+package formats;
+option java_package = "proto";
+option java_outer_classname = "DataFormats";
```

Comment:
How about name it SensorData. since DataFormats doens't give an intuitive describe of what the proto is used to.
The same as the other naming, they are too general, try to give descriptive name.